lib/workers/repository/init/vulnerability.ts

```diff
@@ -174,10 +174,6 @@ export async function detectVulnerabilityAlerts(
           logger.warn({ err }, 'Error generating vulnerability PR notes');
         }
         // TODO: types (#22198)
-        const allowedVersions =
-          datasource === PypiDatasource.id
-            ? `==${val.firstPatchedVersion!}`
```

Comment:
are you sure to drop this condition for pypi?

Reply:
I think this new approach is superior (it's essentially greater than or equal to). Maybe a feature not a refactor though if results can change

Reply:
feature 👍

Reply:
I've rephrased this to feature and will add some extra tests